app/grandchallenge/publications/templates/publications/publication_list.html

```diff
@@ -0,0 +1,51 @@
+{% extends "base.html" %}
+{% load static %}
+{% load publication_extras %}
+{% load guardian_tags %}
+{% load url %}
+
+
+{% block title %}Publications - {{ block.super }}{% endblock %}
+
+{% block breadcrumbs %}
+    <ol class="breadcrumb">
+        <li class="breadcrumb-item active" aria-current="page">Publications</li>
+    </ol>
+{% endblock %}
+
+{% block content %}
+        {% include "grandchallenge/partials/filters.html" with filter=filter filters_applied=filters_applied %}
+    <div>
+        <h2>Publications</h2>
+    </div>
+    <div class="d-flex justify-content-between my-3">
+        <h6 class="ml-1 mt-2">{{ num_publications }} publications | {{ num_citations }} citations </h6>
+        <a class="btn btn-primary" href="{% url 'publications:create' %}" role="button">Add a publication</a>
+    </div>
+        <ul class="list-group">
+            {% for publication in object_list %}
+                <li class="list-group-item p-0">
+                    <div class="py-2 pl-3">
+                        {{ publication.citation|safe }}

```

Comment:
Since the citation is created in the backend, I thought that marking html characters as safe would be ok here, no chance for XSS vulnerability. Right? Alternatively, we could use `striptags` here and miss out on the italics for the journal name, which would also be ok, I guess.

Reply:
Good thinking, and great that you highlight this. We already have a template filter for that, `clean` that you can get from `{% load bleach %}` https://github.com/comic/grand-challenge.org/blob/2a995006b899c9a0f17e0f4ba862b796423c20a1/app/grandchallenge/core/templatetags/bleach.py#L15